### ISSUE TITLE
zlib.dll is not copied to dependent project's output folder

### ISSUE THREAD

**Issue by d-khokhlov** [JohnLangford/vowpal_wabbit#922]
When I install Vowpal Wabbit NuGet package (versions 8.0.0.29 or 8.0.0.30) in some C# class library project A and build it, zlib.dll is copied to output folder. But if another project B in the same solution depends on project A, zlib.dll is not automatically copied to B's output folder during the build. During the execution of B following exception occurs when VW is first used (through some call to A):

```
Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly
'VowpalWabbitCore.dll' or one of its dependencies. The specified module could not be found.
```

If I manually copy zlib.dll to the B's output folder, exception isn't raised and all works fine.

I experienced this in Visual Studio 2015 and didn't test in other versions.

**Comment by JohnLangford** [comment]
@eisber Have you seen this one?

**Comment by JohnLangford** [comment]
@eisber can you take a look?

**Comment by eisber** [comment]
you need to manually add the reference to .zlib in B's output. With the next nuget release this is solved as zlib is linked statically.

**Comment by JohnLangford** [comment]
Thanks @eisber

**Comment by eisber** [comment]
I just published 8.3.3 https://preview.nuget.org/packages/VowpalWabbit/8.3.3

This should fix the issue.
